test/property/arbitraries/canister_methods/inspect_message_method_arb.ts

```diff
@@ -48,14 +48,7 @@ export function InspectMessageMethodArb(
                         : (constraints.methodImplementationLocation ??
                           defaultMethodImplementationLocation);
 
-                const inspectMessageImports =
-                    api === 'functional'
-                        ? ['ic']
-                        : ['msgCaller', 'acceptMessage', 'methodName'];
-                const imports = new Set([
-                    'inspectMessage',
-                    ...inspectMessageImports
-                ]);
+                const imports = new Set(['inspectMessage']);
```

Comment:
I'm suspicious of this change... I'm going to leave this comment here to remind me to come back to it. But where do the msgCaller, acceptMessage, methodName imports from from now?